README.md

```diff
@@ -51,6 +127,24 @@ pytest
 
 > **Note**
 >
-> You will need `pxr` pre-installed into your python environment in order
-> to run these tests. If you have installed USD in the standard manner,
-> it is likely your `PYTHONPATH` will already be extended appropriately.
+> Refer to the [Running](#running) section for the environmental
+> prerequisites to run these tests.
+
+## Project State
+
+This resolver is currently in prototype stage, it has several key
+limitations.
+
```

Comment:
[fix] Add notes that we dont' currently check `managementPolicy` or set a the locale in the context. Also that we haven't currently implemented for performance, and that we presently require a parent python environment.